scripts/modify-layers-pdok-ogcapi.py

```diff
@@ -0,0 +1,172 @@
+import json
+import os
+import shutil
+import requests
+import sys
+
+# python script to modify layers-pdok for the plugin.
+# provide a single parameter to determine behaviour of the script, either:
+# ogcapi `python3 ./scripts/modify-layers-pdok-ogcapi.py ogcapi`: Adds ogcapi test records to layers-pdok.json and creates a copy
+# original `python3 ./scripts/modify-layers-pdok-ogcapi.py original`: Replaces layers-pdok.json with the original copy and removes the copy
+
+
+def extend_layer_pdok_ogcapi(urls_oaf = [], urls_oat = []):
+        layers_pdok = []
+        layers_pdok = retrieve_layers_from_oat_endpoint(urls_oat)
+        layers_pdok.extend(retrieve_layers_from_oaf_endpoint(urls_oaf))
+        return layers_pdok
+    
+def retrieve_layers_from_oat_endpoint(urls = []):
```

Comment:
styling: dubbele enter

Reply:
Deze komt nog in het hele script voor. Tussen classes en functions in de script/package scope horen twee enters.

Reply:
Als het goed is is de styling nu overal gefixt. Ik heb ook black (python codeformatter van de plugin) in deze file gerunt om de code consistent te krijgen, ondanks dat dit een script is